### DISCUSSION TITLE
Can you just type something here once if you'd be interested in joining the server?

### DISCUSSION THREAD

**Discussion by lesleyespire** [jacobfortune/Songs-Of-War-Addon#2] [General]
Can you just type something here once if you'd be interested in joining the server? Just so I can get a rough idea of how many people would join, I already have about 20 people but I've only just started.

**Comment by Resistantsmile** [comment]
Hello

**Comment by lesleyespire** [comment]
Hello, I actually have a much better server and we're working on the best SoW addon with an epic map if you'd like to join. I'll send the IP once we launch beta. I'm Solos btw, not actually Pyrobyte.
________________________________

**Reply by Resistantsmile** [reply]
Can I join the server?

**Comment by lesleyespire** [comment]
Yes you are welcome to, thanks, we have a discord invite and I'll post the ip later here

________________________________

**Comment by lesleyespire** [comment]
<https://discord.gg/tgYhJryaCM>
discord.gg<https://discord.gg/tgYhJryaCM>
[X]<https://discord.gg/tgYhJryaCM>


On 16 Jan 2024, at 2:38 pm, Solos Sendaris ***@***.***> wrote:

﻿
Yes you are welcome to, thanks, we have a discord invite and I'll post the ip later here

________________________________

**Comment by Mercer-BloodSong** [comment]
Nothing wrong with me joining ja?

**Reply by lesleyespire** [reply]
You're welcome to join! Unfortunately not many people are active there anymore so its almost completely dead, (I'd like to revive it during the summer possibly)

**Reply by Mercer-BloodSong** [reply]
Well that would great thanks bud

**Reply by lesleyespire** [reply]
Yeah, I was thinking, since the squared media reboot seems to be going ahead, there's a lot of potential for the fanbase to be active again

**Reply by Mercer-BloodSong** [reply]
so very true
